src/Utility.js

```diff
@@ -92,6 +92,18 @@ const __class = lang.setObject('crm.Utility', lang.mixin({}, Utility, {
     }
     return id;
   },
+  trimText: function trimText(text = {}, wordCount = {}) {
```

Comment:
I would default text and wordCount params to their correct value type. text = '' and wordCount = 0 as defaults.